src/sourmash_plugin_pangenomics.py

```diff
@@ -328,7 +335,11 @@ def pangenome_createdb_main(args):
 
 # Chunk function to limit the memory used by the hash_count dict and list
 def write_chunk(chunk, output_file):
-    with open(output_file, "a", newline="") as csvfile:
+    if output_file.endswith('.gz'):
```

Comment:
this looks good - not sure you need the `--gzip` option after all? It doesn't seem to add much :).